apps/mosaico/src/MosaicoServer.purs

```diff
@@ -17,15 +17,11 @@ import Simple.JSON (write)
 
 type Props =
   { mainContent :: MainContent
-  , mostReadArticles :: Array ArticleStub
+  , mostReadArticles :: JSX
```

Comment:
I don't like this. `JSX` is opaque and using it scatters it all over `Main.purs` what's really rendered in there and it opens up possibilities for bugs since `JSX` could be just anything that's not related to a list of articles whatsoever. If it won't be an `Array ArticleStub` then introduce some other new type for this.

`Main.purs` is overloaded enough even as is. Seeding data types in it is lighter than calling renders.

Reply:
Ah yes it's from a previous iteration where I was breaking everything